blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js

```diff
@@ -670,7 +675,8 @@ export default function SimulationScreen ({ open, close }) {
   }
 
   const typography1 = 'SOMETHING WENT WRONG. Please Check The Simulation Parameters.'
-  const typography2 = 'SOMETHING WENT WRONG. Please Check The Simulation Parameters And ' + process.env.REACT_APP_DIAGRAM_NAME + '.'
+  const typography2 = 'PLease Wait The Graph is Rendering with' + process.env.REACT_APP_DIAGRAM_NAME + '.'
```

Comment:
```suggestion
  const typography2 = 'Please Wait. The Graph is Rendering with ' + process.env.REACT_APP_DIAGRAM_NAME + '...'
```
Small corrections:

- Please, not PLease
- full stop after Wait
- space after with